docs/docker-example/Dockerfile

```diff
@@ -10,19 +10,18 @@ RUN apt-get install -y python3-pip
 RUN python -m pip install --upgrade pip
 RUN apt-get install -y gdb
 RUN apt-get install -y lldb
-RUN apt-get install -y emacs
 RUN apt-get install -y vim
-# 2) Instal optional packages, uncomment/add as you see fit.
-# RUN apt-get install -y valgrind
-# RUN apt-get install -y libeigen3-dev
-# RUN apt-get install -y clang-format
-# RUN python -m pip install wheel
-# RUN python -m pip install numpy
-# RUN python -m pip install torch==1.6.0+cpu torchvision==0.7.0+cpu -f https://download.pytorch.org/whl/torch_stable.html
-# RUN git clone https://github.com/onnx/tutorials.git
+# 2) Instal optional packages, comment/uncomment/add as you see fit.
+RUN apt-get install -y valgrind
+RUN apt-get install -y libeigen3-dev
+RUN apt-get install -y clang-format
+RUN python -m pip install wheel
+RUN python -m pip install numpy
+RUN python -m pip install torch==1.6.0+cpu torchvision==0.7.0+cpu -f https://download.pytorch.org/whl/torch_stable.html
+RUN git clone https://github.com/onnx/tutorials.git
 # Install clang-12.
```

Comment:
Prob. best to avoid naming the exact version of clang as that will change.

Reply:
good point, the install is actually agnostic of version.